### PR TITLE
Fix incorrect Markdown

### PR DESCRIPTION
**Pull request by lex111**
https://developer.mozilla.org/en-US/docs/Web/CSS/content

image

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet/WeakSet

image

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/weakset/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ ws.has(foo);    // false, foo has been removed
 ws.has(bar);    // true, bar is retained
 ```
 
-Note that `foo !== bar`. While they are similar objects, \*they are not **the same object\***. And so they are both added to the set.
```
Comment (lex111) [inline files/en-us/web/javascript/reference/global_objects/weakset/index.md:100]: Apparently, the converter from HTML to MD escapes `*` characters, which causes italic inside bold to be rendered incorrectly.

Comment (lex111) [reply files/en-us/web/javascript/reference/global_objects/weakset/index.md:100]: Created issue in yari repo https://github.com/mdn/yari/issues/4648

Comment (teoli2003) [reply files/en-us/web/javascript/reference/global_objects/weakset/index.md:100]: cc/ @wbamberg. I think we should do a check for these cases when we convert more.

**Review by teoli2003 — APPROVED** [review]
Good catch. Thanks.

**Comment by wbamberg** [comment]
Thanks for finding this, @lex111 , and for the ping, @teoli2003 . I've been through all the pages in web/api where an `<em>` contains a `<strong>`. In most cases this is getting converted properly, but there are three pages where it's getting mangled:

content/files/en-us/web/api/mswriteprofilermark/index.html
content/files/en-us/web/api/validitystate/badinput/index.html
content/files/en-us/web/api/validitystate/index.html

I'll file a PR to fix them in the HTML.

**Pull request merged**